### PR TITLE
pages.id/*: update Indonesian translations

### PR DESCRIPTION
**Pull request by reinhart1010**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

This PR provides miscellaneous fixes to Indonesian translations, mostly by request of @sebastiaanspeck.

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]

File: pages.id/common/rm.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
 
-- Menghapus direktori dan semua subdirektorinya secara [r]ekursif:
+- Hapus file dengan mode [v]erbose, mencetak pesan untuk setiap file yang terhapus:
 
-`rm -r {{alamat/ke/direktori}}`
+`rm -v {{alamat/ke/file1 alamat/ke/file2 ...}}`
+
+- Hapus direktori dan semua subdirektorinya secara [r]ekursif:
+
+`rm -{{alamat/ke/file1 alamat/ke/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:25]:
```suggestion
`rm -r {{jalan/menuju/file_atau_direktori1 jalan/menuju/file_atau_direktori2 ...}}`
```

File: pages.id/common/rm.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+- Hapus banyak file secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
 
-`rm -v {{alamat/ke/directori/*}}`
+`rm -i {{alamat/ke/file1 alamat/ke/file2 ...}}`
 
-- Menghapus direktori dan semua subdirektorinya secara [r]ekursif:
+- Hapus file dengan mode [v]erbose, mencetak pesan untuk setiap file yang terhapus:
 
-`rm -r {{alamat/ke/direktori}}`
+`rm -v {{alamat/ke/file1 alamat/ke/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:21]:
```suggestion
`rm -v {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```

File: pages.id/common/rm.md [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+- Hapus file dari lokasi manapun dengan mengabaikan nama-nama file yang tidak ditemukan:
 
-`rm -i {{(beberapa)_berkas}}`
+`rm -f {{alamat/ke/file1 alamat/ke/file2 ...}}`
 
-- Menghapus berkas dengan mode verbose, mencetak pesan untuk setiap berkas yang terhapus:
+- Hapus banyak file secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
 
-`rm -v {{alamat/ke/directori/*}}`
+`rm -i {{alamat/ke/file1 alamat/ke/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:17]:
```suggestion
`rm -i {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```

File: pages.id/common/rm.md [hunk 4]
```diff
@@ -1,20 +1,25 @@
...
+- Hapus file dari lokasi manapun:
 
-`rm {{alamat/ke/berkas1 alamat/ke/berkas2 ...}}`
+`rm {{alamat/ke/file1 alamat/ke/file2 ...}}`
 
-- Menghapus banyak berkas secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
+- Hapus file dari lokasi manapun dengan mengabaikan nama-nama file yang tidak ditemukan:
 
-`rm -i {{(beberapa)_berkas}}`
+`rm -f {{alamat/ke/file1 alamat/ke/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:13]:
```suggestion
`rm -f {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```

File: pages.id/common/rm.md [hunk 5]
```diff
@@ -1,20 +1,25 @@
...
-> Menghapus berkas atau direktori.
+> Hapus file atau direktori.
+> Lihat juga: `rmdir`.
 > Informasi lebih lanjut: <https://www.gnu.org/software/coreutils/rm>.
 
-- Menghapus berkas dari lokasi manapun:
+- Hapus file dari lokasi manapun:
 
-`rm {{alamat/ke/berkas1 alamat/ke/berkas2 ...}}`
+`rm {{alamat/ke/file1 alamat/ke/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:9]:
```suggestion
`rm {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

**Review by gutjuri — APPROVED** [review]

**Comment by reinhart1010** [comment]
> Thanks for your contribution. I have some suggestions for this page.
> 
> Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

@kbdharun At the same time I was planning to change the official `path/to/file` translation from `jalan/menuju/file` to `jalan/menuju/berkas` as [`berkas` is the sanctioned Indonesian term for computer files](https://kbbi.kemdikbud.go.id/entri/berkas).

**Comment by reinhart1010** [comment]
> @kbdharun At the same time I was planning to change the official `path/to/file` translation from `jalan/menuju/file` to `jalan/menuju/berkas` as [`berkas` is the sanctioned Indonesian term for computer files](https://kbbi.kemdikbud.go.id/entri/berkas).

This is covered in #12083, but this PR still uses the old `jalan/menuju/file`.

**Comment by kbdharun** [comment]
> This is covered in https://github.com/tldr-pages/tldr/pull/12083, but this PR still uses the old jalan/menuju/file.

Thanks for the clarification, the change LGTM (feel free to use the native terms here too).

File: pages.id/common/cat.md [hunk 6]
```diff
@@ -1,16 +1,24 @@
...
 
-> Mencetak dan menggabungkan berkas.
+> Cetak dan menggabungkan berkas.
 > Informasi lebih lanjut: <https://www.gnu.org/software/coreutils/cat>.
 
-- Mencetak konten berkas ke keluaran standar:
+- Cetak konten berkas menuju `stdout`:
 
-`cat {{berkas}}`
+`cat {{jalan/menuju/file}}`
```
Comment (kbdharun) [inline pages.id/common/cat.md:8]:
```suggestion
`cat {{jalan/menuju/berkas}}`
```

File: pages.id/common/cat.md [hunk 7]
```diff
@@ -1,16 +1,24 @@
...
 
-- Menggabungkan konten beberapa berkas ke berkas tujuan:
+- Gabungkan konten beberapa berkas ke berkas tujuan:
 
-`cat {{berkas1 berkas2 ...}} > {{berkas_tujuan}}`
+`cat {{jalan/menuju/file1 jalan/menuju/file2 ...}} > {{jalan/menuju/file_tujuan}}`
```
Comment (kbdharun) [inline pages.id/common/cat.md:12]:
```suggestion
`cat {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}} > {{jalan/menuju/berkas_tujuan}}`
```

File: pages.id/common/cat.md [hunk 8]
```diff
@@ -1,16 +1,24 @@
...
 
-- Menambahkan konten beberapa berkas ke berkas tujuan:
+- Tambahkan konten beberapa berkas ke berkas tujuan:
 
-`cat {{berkas1 berkas2 ...}} >> {{berkas_tujuan}}`
+`cat {{jalan/menuju/file1 jalan/menuju/file2 ...}} >> {{jalan/menuju/file_tujuan}}`
```
Comment (kbdharun) [inline pages.id/common/cat.md:16]:
```suggestion
`cat {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}} >> {{jalan/menuju/berkas_tujuan}}`
```

File: pages.id/common/cat.md [hunk 9]
```diff
@@ -1,16 +1,24 @@
...
+
+- Salin isi suatu file menuju file tujuan tanpa proses buffering:
+
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
+
+- Tulis isi `stdin` menuju suatu file:
+
+`cat - > {{jalan/menuju/file}}`
```
Comment (kbdharun) [inline pages.id/common/cat.md:24]:
```suggestion
`cat - > {{jalan/menuju/berkas}}`
```

File: pages.id/common/mongod.md [hunk 10]
```diff
@@ -1,16 +1,20 @@
...
 
-> Server database MongoDB.
+> Peladen basis data MongoDB.
 > Informasi lebih lanjut: <https://docs.mongodb.com/manual/reference/program/mongod>.
 
-- Menentukan berkas konfigurasi:
+- Tentukan direktori penyimpanan basis data (default: `/data/db` dalam Linux dan macOS, `C:\data\db` dalam Windows):
 
-`mongod --config {{nama_berkas}}`
+`mongod --dbpath {{jalan/menuju/directory}}`
```
Comment (kbdharun) [inline pages.id/common/mongod.md:8]:
```suggestion
`mongod --dbpath {{jalan/menuju/direktori}}`
```

File: pages.id/common/mongod.md [hunk 11]
```diff
@@ -1,16 +1,20 @@
...
 
-- Menentukan port yang digunakan:
+- Tentukan berkas konfigurasi basis data:
+
+`mongod --config {{jalan/menuju/file}}`
```
Comment (kbdharun) [inline pages.id/common/mongod.md:12]:
```suggestion
`mongod --config {{jalan/menuju/berkas}}`
```

File: pages.id/common/rm.md [hunk 12]
```diff
@@ -1,20 +1,25 @@
...
-> Menghapus berkas atau direktori.
+> Hapus file atau direktori.
+> Lihat juga: `rmdir`.
 > Informasi lebih lanjut: <https://www.gnu.org/software/coreutils/rm>.
 
-- Menghapus berkas dari lokasi manapun:
+- Hapus file dari lokasi manapun:
 
-`rm {{alamat/ke/berkas1 alamat/ke/berkas2 ...}}`
+`rm {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:9]:
```suggestion
`rm {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}}`
```

File: pages.id/common/rm.md [hunk 13]
```diff
@@ -1,20 +1,25 @@
...
 
-- Menghapus banyak berkas secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
+- Hapus file dari lokasi manapun dengan mengabaikan nama-nama file yang tidak ditemukan:
 
-`rm -i {{(beberapa)_berkas}}`
+`rm -f {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:13]:
```suggestion
`rm -f {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}}`
```

File: pages.id/common/rm.md [hunk 14]
```diff
@@ -1,20 +1,25 @@
...
 
-- Menghapus berkas dengan mode verbose, mencetak pesan untuk setiap berkas yang terhapus:
+- Hapus banyak file secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
 
-`rm -v {{alamat/ke/directori/*}}`
+`rm -i {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:17]:
```suggestion
`rm -i {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}}`
```

File: pages.id/common/rm.md [hunk 15]
```diff
@@ -1,20 +1,25 @@
...
 
-- Menghapus direktori dan semua subdirektorinya secara [r]ekursif:
+- Hapus file dengan mode [v]erbose, mencetak pesan untuk setiap file yang terhapus:
 
-`rm -r {{alamat/ke/direktori}}`
+`rm -v {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
+
+- Hapus direktori dan semua subdirektorinya secara [r]ekursif:
+
+`rm -r {{jalan/menuju/file_atau_direktori1 jalan/menuju/file_atau_direktori2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:25]:
```suggestion
`rm -r {{jalan/menuju/berkas_atau_direktori1 jalan/menuju/berkas_atau_direktori2 ...}}`
```

File: pages.id/common/mv.md [hunk 16]
```diff
@@ -1,24 +1,32 @@
 # mv
 
-> Memindah atau menamai-ulang file dan direktori.
+> Pindahkan atau namai-ulang file dan direktori.
```
Comment (kbdharun) [inline pages.id/common/mv.md:3]:
```suggestion
> Pindahkan atau namai-ulang berkas dan direktori.
```

File: pages.id/common/mv.md [hunk 17]
```diff
@@ -1,24 +1,32 @@
...
 > Informasi lebih lanjut: <https://www.gnu.org/software/coreutils/mv>.
 
-- Memindahkan file ke lokasi yang baru:
+- Pindahkan file ke lokasi yang baru:
```
Comment (kbdharun) [inline pages.id/common/mv.md:6]:
```suggestion
- Pindahkan berkas ke lokasi yang baru:
```

File: pages.id/common/mv.md [hunk 18]
```diff
@@ -1,24 +1,32 @@
...
 
 `mv {{jalan/menuju/sumber}} {{jalan/menuju/tujuan}}`
 
-- Memindah tanpa meminta konfirmasi sebelum menimpa file yang sudah ada:
+- Pindahkan file atau direktori ke dalam direktori lain yang telah ada:
+
+`mv {{jalan/menuju/sumber}} {{jalan/menuju/direktori}}`
+
+- Pindahkan file majemuk ke dalam direktori lain yang telah ada, dengan menyimpan nama masing-masing file secara utuh:
```
Comment (kbdharun) [inline pages.id/common/mv.md:14]:
```suggestion
- Pindahkan berkas atau direktori ke dalam direktori lain yang telah ada:

`mv {{jalan/menuju/sumber}} {{jalan/menuju/direktori}}`

- Pindahkan berkas majemuk ke dalam direktori lain yang telah ada, dengan menyimpan nama masing-masing file secara utuh:
```

File: pages.id/common/mv.md [hunk 19]
```diff
@@ -1,24 +1,32 @@
...
+
+`mv {{jalan/menuju/sumber1 jalan/menuju/sumber2 ...}} {{jalan/menuju/direktori}}`
+
+- Pindahkan tanpa meminta konfirmasi sebelum menimpa file yang sudah ada:
 
 `mv -f {{jalan/menuju/sumber}} {{jalan/menuju/tujuan}}`
 
-- Meminta konfirmasi sebelum menimpa file yang sudah ada, apapun file permissions-nya:
+- Minta konfirmasi sebelum menimpa file yang sudah ada, tanpa memerhatikan hak akses hedua file tersebut:
```
Comment (kbdharun) [inline pages.id/common/mv.md:22]:
```suggestion
- Minta konfirmasi sebelum menimpa berkas yang sudah ada, tanpa memerhatikan hak akses hedua file tersebut:
```

File: pages.id/common/mv.md [hunk 20]
```diff
@@ -1,24 +1,32 @@
...
 
 `mv -i {{jalan/menuju/sumber}} {{jalan/menuju/tujuan}}`
 
 - Jangan menimpa file yang sudah ada di direktori tujuan:
 
 `mv -n {{jalan/menuju/sumber}} {{jalan/menuju/tujuan}}`
 
-- Memindahkan file dalam mode verbose, menampilkan file-file yang dipindahkan:
+- Pindahkan file dalam mode [v]erbose, tampilkan file-file yang dipindahkan:
```
Comment (kbdharun) [inline pages.id/common/mv.md:30]:
```suggestion
- Pindahkan berkas dalam mode [v]erbose, tampilkan berkas-berkas yang dipindahkan:
```

File: pages.id/common/rm.md [hunk 21]
```diff
@@ -1,20 +1,25 @@
 # rm
 
-> Menghapus berkas atau direktori.
+> Hapus file atau direktori.
```
Comment (kbdharun) [inline pages.id/common/rm.md:3]:
```suggestion
> Hapus berkas atau direktori.
```

File: pages.id/common/rm.md [hunk 22]
```diff
@@ -1,20 +1,25 @@
...
+> Lihat juga: `rmdir`.
 > Informasi lebih lanjut: <https://www.gnu.org/software/coreutils/rm>.
 
-- Menghapus berkas dari lokasi manapun:
+- Hapus file dari lokasi manapun:
```
Comment (kbdharun) [inline pages.id/common/rm.md:7]:
```suggestion
- Hapus berkas dari lokasi manapun:
```

File: pages.id/common/rm.md [hunk 23]
```diff
@@ -1,20 +1,25 @@
...
 
-`rm {{alamat/ke/berkas1 alamat/ke/berkas2 ...}}`
+`rm {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}}`
 
-- Menghapus banyak berkas secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
+- Hapus file dari lokasi manapun dengan mengabaikan nama-nama file yang tidak ditemukan:
```
Comment (kbdharun) [inline pages.id/common/rm.md:11]:
```suggestion
- Hapus berkas dari lokasi manapun dengan mengabaikan nama-nama file yang tidak ditemukan:
```

File: pages.id/common/rm.md [hunk 24]
```diff
@@ -1,20 +1,25 @@
...
 
-`rm -i {{(beberapa)_berkas}}`
+`rm -f {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}}`
 
-- Menghapus berkas dengan mode verbose, mencetak pesan untuk setiap berkas yang terhapus:
+- Hapus banyak file secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
```
Comment (kbdharun) [inline pages.id/common/rm.md:15]:
```suggestion
- Hapus banyak berkas secara [i]nteraktif, dengan meminta konfirmasi sebelum setiap penghapusan:
```

File: pages.id/common/rm.md [hunk 25]
```diff
@@ -1,20 +1,25 @@
...
 
-`rm -v {{alamat/ke/directori/*}}`
+`rm -i {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}}`
 
-- Menghapus direktori dan semua subdirektorinya secara [r]ekursif:
+- Hapus file dengan mode [v]erbose, mencetak pesan untuk setiap file yang terhapus:
```
Comment (kbdharun) [inline pages.id/common/rm.md:19]:
```suggestion
- Hapus berkas dengan mode [v]erbose, mencetak pesan untuk setiap file yang terhapus:
```

File: pages.id/common/rm.md [hunk 26]
```diff
@@ -1,20 +1,25 @@
...
 
-`rm -r {{alamat/ke/direktori}}`
+`rm -v {{jalan/menuju/file1 jalan/menuju/file2 ...}}`
```
Comment (kbdharun) [inline pages.id/common/rm.md:21]:
```suggestion
`rm -v {{jalan/menuju/berkas1 jalan/menuju/berkas2 ...}}`
```

File: pages.id/linux/pacman.md [hunk 27]
```diff
@@ -2,20 +2,25 @@
...
-- Instal paket baru:
+- Pasang suatu paket baru:
 
 `sudo pacman -S {{paket}}`
 
 - Hapus paket beserta dependensinya:
 
 `sudo pacman -Rs {{paket}}`
 
+- Cari pangkalan data untuk nama-nama paket yang mengandung suatu file secara spesifik:
```
Comment (kbdharun) [inline pages.id/linux/pacman.md:20]:
```suggestion
- Cari pangkalan data untuk nama-nama paket yang mengandung suatu berkas secara spesifik:
```

File: pages.id/linux/pacman.md [hunk 28]
```diff
@@ -2,20 +2,25 @@
...
+
+`pacman -F "{{nama_file}}"`
```
Comment (kbdharun) [inline pages.id/linux/pacman.md:22]:
```suggestion
`pacman -F "{{nama_berkas}}"`
```

File: pages.id/common/ab.md [hunk 29]
```diff
@@ -22,3 +22,7 @@
 - Alokasikan wak[t]u maksimum (dalam hitungan detik) untuk mengujinya:
 
 `ab -t {{60}} {{url}}`
+
+- Simpan hasil pengujian menuju suatu file CSV:
+
+`ab -e {{jalan/menuju/file.csv}}`
```
Comment (kbdharun) [inline pages.id/common/ab.md:28]:
```suggestion
`ab -e {{jalan/menuju/berkas.csv}}`
```

File: pages.id/common/ab.md [hunk 30]
```diff
@@ -22,3 +22,7 @@
 - Alokasikan wak[t]u maksimum (dalam hitungan detik) untuk mengujinya:
 
 `ab -t {{60}} {{url}}`
+
+- Simpan hasil pengujian menuju suatu file CSV:
```
Comment (kbdharun) [inline pages.id/common/ab.md:26]:
```suggestion
- Simpan hasil pengujian menuju suatu berkas CSV:
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution, merging this PR now (made the requested changes), so feel free to fix any issues in a new PR.

**Pull request merged**